src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/Views/Home/Index.cshtml

```diff
@@ -6,3 +6,8 @@
     <h1 class="display-4">Welcome</h1>
     <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
 </div>
+    @* #if(GenerateApiOrGraph)
```

Comment:
FYI this doesn't work. The generated templates just has this section verbatim.

Reply:
I tried fixing this with https://github.com/dotnet/aspnetcore/pull/24167/commits/3b093a88569e418fdc3c45340cbc0e26faa46085. It's working as intended but doesn't seem like the right syntax.